### DISCUSSION TITLE
Theme color swap

### DISCUSSION THREAD

**Discussion by JUNEDSK** [TDRA-ae/aegov-dls#26] [Q&A]
Hi Team,

As I have added the color inside the tailwind.config.js file I wanted more clarification on what would be the next step to apply my colors to all over the theme also do I need to replace the colors inside the dist/style.css 

![Screenshot 2024-03-19 133200](https://github.com/TDRA-ae/aegov-dls/assets/36602099/1ac802e4-eece-49b7-bfcf-e258b14f81c0)

**Comment by moiz-conceptlz** [comment]
Hello @JUNEDSK 

Let me first clarify that the primary colour is by default set to AE-GOLD and the the secondary colour is by default set to AE-Black as mentioned on https://designsystem.gov.ae/guidelines/colour-system

If you intend to use the AE-GOLD colour for your website, you do not need to add the colour in your tailwind.config.js 

The `@aegov/design-system` plugin already has default settings - which you can refer to on https://designsystem.gov.ae/docs/extending-the-configuration. You will notice that the colour definition of "**primary**" already points to the AE-GOLD swatch.

Hence, to ever use the primary colour within your layouts, you may simply add the <type>-primary-<swatch> to the HTML element. Example, if you want to change the colour of the text to AE-GOLD 700, then you would add the class `text-primary-700`. Similarly, if you need to add a background colour as the AE-GOLD 100, you would add the class `bg-primary-100`. If you ever need to apply opacity to a background colour, you would add the class like the following:
```
<div class="bg-primary-500/[.06]">
</div>
```
or if you need the complete div to have an opacity, then you would do this:
```
<div class="bg-primary-500 opacity-60">
</div>
```

Refer to https://tailwindcss.com/docs/background-color or https://tailwindcss.com/docs/text-color for more details.
